src/layout/internal/TabbingLayoutContainer.ts

```diff
@@ -44,6 +44,30 @@ export default class TabbingLayoutContainer extends AParentLayoutContainer<ITabb
       }
       return true;
     }, null, true);
+
+    if(this.options.fixed) {
+      this.header.classList.add('frozen');
```

Comment:
Why is this CSS class called `frozen` and not `fixed`, since it gets active when setting `options.fixed`?